### PR TITLE
Add debugging to the codegen process

### PR DESCRIPTION
**Pull request by thockin**
This was useful when debugging a different problem.

### CODE REVIEW

**Comment by thockin** [comment]
@ixdy - this should be a relatively easy one, despite the scary name.

File: pkg/api/unversioned/types.go [hunk 1]
```diff
@@ -23,8 +23,6 @@ package unversioned
 // TypeMeta describes an individual object in an API response or request
 // with strings representing the type of the object and its API schema version.
 // Structures that are versioned or persisted should inline TypeMeta.
-//
-// +k8s:deepcopy-gen=false
```
Comment (ixdy) [inline pkg/api/unversioned/types.go]: were these removals intentional?

Comment (thockin) [reply pkg/api/unversioned/types.go]: Yes, they cause spurious rebuilds but generate no output (so the dep is never satisfied).

**Comment by thockin** [comment]
ping

**Pull request merged**